ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-commands.sh

```diff
@@ -61,9 +61,8 @@ if ! git clone -b "$DESTINATION_BRANCH" "$repo_url" ; then
         exit 1
     fi
 
-    echo "INFO Pushing the following commits to $DESTINATION_BRANCH to origin"
-    git --no-pager log --pretty=oneline origin/"$DESTINATION_BRANCH"..HEAD
-    if ! git push origin $DESTINATION_BRANCH; then
+    echo "INFO Pushing to new branch $DESTINATION_BRANCH"
+    if ! git push --set-upstream origin "$DESTINATION_BRANCH" ; then
```

Comment:
Please mention why you are making this change in your commit message.

I also don't understand why you need the `--set-upstream`. Why is that needed given it is an ephemeral config change?

Reply:
Updated the commit message and removed the `--set-upstream` given what you said. I was able to run this locally with no errors.